hadoop-ozone/common/src/main/proto/OzoneManagerProtocol.proto

```diff
@@ -695,6 +695,8 @@ message KeyArgs {
 
     // This will be set by leader OM in HA and update the original request.
     optional FileEncryptionInfoProto fileEncryptionInfo = 15;
+    // This is a key list to support the batch operation of keys.
+    repeated string keyNames = 16;
```

Comment:
Why not remain KeyArgs as for single key?

Reply:
Yes, We can avoid changing it here，by use list of KeyArgs.